frontend/src/metabase/search/components/InfoText.tsx

```diff
@@ -12,11 +12,10 @@ import { PLUGIN_COLLECTIONS } from "metabase/plugins";
 import { getTranslatedEntityName } from "metabase/nav/utils";
 
 import type { Collection } from "metabase-types/api";
+import type { WrappedResult } from "metabase/search/util/wrapped-result-type";
 import type TableType from "metabase-lib/metadata/Table";
 
 import { CollectionBadge } from "./CollectionBadge";
-import type { WrappedResult } from "./types";
```

Comment:
did you mean to delete the `components/types.ts` file ?

Reply:
Yep, that was deliberate. I figured that we would eventually move all types and things into `search/types/...` or `search/types.ts` to keep things relatively clean